.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,39 @@
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+
+jobs:
+  static-analysis-psalm:
+    name: "Static Analysis with Psalm"
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+
+    steps:
+      - name: "Checkout code"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php-version }}"
+          tools: cs2pr
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v1"
+        with:
+          path: "~/.composer/cache"
+          key: "${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        run: "composer install --no-interaction --no-progress --no-suggest"
+
+      - name: "Psalm"
+        run: "vendor/bin/psalm --output-format=checkstyle | cs2pr"
```

Comment:
psalm has native support, no need for `cs2pr`
`vendor/bin/psalm --show-info=false --stats --output-format=github --threads=4`

Reply:
Fixed